docs/user-guide/src/version_support.md

```diff
@@ -83,16 +86,16 @@ The table describes which branches/image-tags are tested in each periodic CI tes
 | ------------------------------------------------ | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
 | daily_main_integration_test_ubuntu/centos        | main         | main        | main           | latest         | latest      | latest     |
 | daily_main_e2e_integration_test_ubuntu/centos    | main         | main        | main           | latest         | latest      | latest     |
+| daily_release-1-7_integration_test_ubuntu/centos | release-1.7  | release-1.7 | release-0.6    | v0.6.0         | latest      | latest     |
```

Comment:
This table seems outdated we are now using e2e tests as main tests can we update the table before merging the PR

Reply:
the job naming also changed recently

Reply:
Sure. Does it look ok now?